src/main/java/at/hannibal2/skyhanni/features/inventory/QuiverNotification.kt

```diff
@@ -0,0 +1,24 @@
+package at.hannibal2.skyhanni.features.inventory
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.TitleManager
+import at.hannibal2.skyhanni.events.LorenzChatEvent
+import at.hannibal2.skyhanni.utils.SoundUtils
+import at.hannibal2.skyhanni.utils.StringUtils.matchMatcher
+import at.hannibal2.skyhanni.utils.StringUtils.removeColor
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import kotlin.time.Duration.Companion.seconds
+
+object QuiverNotification {
+    private val quiverChatPattern by RepoPattern.pattern("inventory.quiver.chat.low", "You only have (?<arrowsLeft>.*) arrows left in your Quiver!")
```

Comment:
Please include color codes in the pattern, and then dont use  `removeColor` below.